src/components/PopoverWithMeasuredContent.js

```diff
@@ -53,14 +63,14 @@ class PopoverWithMeasuredContent extends Component {
     constructor(props) {
         super(props);
 
+        this.popoverWidth = this.props.popoverDimensions.width;
+        this.popoverHeight = this.props.popoverDimensions.height;
+
         this.state = {
-            isContentMeasured: false,
+            isContentMeasured: !!(this.popoverWidth > 0 && this.popoverHeight > 0),
```

Comment:
Conditionals always return boolean.
```suggestion
            isContentMeasured: this.popoverWidth > 0 && this.popoverHeight > 0,
```

Reply:
This was autocorrected by `eslint`.